sympy/polys/matrices/domainmatrix.py

```diff
@@ -375,10 +377,10 @@ def from_dict_sympy(cls, nrows, ncols, elemsdict, **kwargs):
         idx = 0
         items_dict = {}
         for i, row in elemsdict.items():
-                items_dict[i] = {}
-                for j in row:
-                    items_dict[i][j] = items_domain[idx]
-                    idx += 1
+            items_dict[i] = {}
```

Comment:
@oscarbenjamin I use intellij and I have a habit of hitting the formatting shortcut a lot when I code. Did my formatter find and fix an indentation error here? If so, do you mind fixes like this bundled into feature commits, or do you like to make a seperate bug ticket/commit?

Reply:
There is no error here but this change has corrected the indentation from a style perspective. This one is fine but best not to bundle any more in. It's better to separate changes that touch lots of lines of code but don't change behaviour from changes that touch few lines of code and have significant implications.